src/util.js

```diff
@@ -14,12 +14,17 @@ const focusableElementsList = [
     '*[contenteditable]'
 ];
 
-function createTrapBoundary() {
-    const div = document.createElement('div');
-    div.setAttribute('tabindex', '0');
-    div.className = 'keyboard-trap-boundary';
+let trapBoundary;
+if (typeof document !== "undefined") {
```

Comment:
Thanks for adding this check in the util. This makes me realize I need a similar check elsewhere in other makeup module utils.